.editorconfig

```diff
@@ -0,0 +1,22 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
```

Comment:
just FYI (not applicable there) - trailing whitespaces are sometimes necessary in AsciiDoc files (because it's grammar is pretty bad in many corner cases).

Reply:
@tisnik Also in `.vimrc` and various scripts, where strings are written outside of quotes and you want to write a string ending with whitespace.

In general, however, I assume you don't need trailing whitespace and I've seen a lot of it in the Markdown files.